applications/asset_tracker/src/main.c

```diff
@@ -488,6 +492,20 @@ static void on_user_association_req(const struct nrf_cloud_evt *p_evt)
 	}
 }
 
+/**@brief Callback for data received event from nRF Cloud. */
+static void on_data_received(const struct nrf_cloud_evt *p_evt)
+{
+	if (memcmp(p_evt->param.data.ptr, CLOUD_LED_ON_STR,
+		strlen(CLOUD_LED_ON_STR)) == 0) {
```

Comment:
Could use `sizeof CLOUD_LED_ON_STR - 1` to do this compile time instead of runtime